definitions/variable/industry/production.yaml

```
@@ -0,0 +1,15 @@
+- Production|Iron and Steel|{Iron Commodity} [{Accounting Type}]:
+    description: production of {Iron Commodity} in {Accounting Type}
```

Please add "given as {Accounting Type}" to all variables where the Accounting-Type is used
```suggestion
    description: production of {Iron Commodity} given as {Accounting Type}
```